### ISSUE TITLE
Widget for mensa menu

### ISSUE THREAD

**Issue by backes** [uds-se/uni-app-android#4]
It would be really nice to have a widget which shows the current mensa menu.
This is somewhat related to #3, as it requires RemoteViews for the mensa menu.
This means that it should be easy to implement once #3 is done.

**Comment by backes** [comment]
Done in >= 1.5.
